client/src/components/employer-dashboard-component/employer-dashboard-filter/index.jsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Card,
+  CardContent,
+  TextField,
+  InputAdornment,
+  Typography,
+} from "@mui/material";
+import SearchIcon from "@mui/icons-material/Search";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider, DatePicker } from "@mui/x-date-pickers";
+
+function EmployerDashboardFilter() {
+  return (
+    <Card sx={{ width: 240, height: "fit-content", marginLeft: 2 }}>
```

Comment:
Setting up the onChange for each input field type would be great. As the query variables would be ready to go when it comes to connecting them
You can take a look at Kevin's `client/src/components/job-lead-dashboard-component/job-lead-dashboard-filters/index.jsx` Just using useState